Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8-alpine
```

Comment:
Validating this part internally for software licensing.

Reply:
Alpine for smaller size. Could use ```3.8-slim``` (Debian Buster) instead